frontend/src/views/DriveFormView.vue

```diff
@@ -155,6 +155,29 @@
                   :class="{ 'is-invalid': errors['endMileage'] }"
                 >
               </div>
+              <div class="form-group">
+                <label for="driveHash">{{ $t('drive_form.drive_hash') }}</label>
+                <input
+                  value="TODO"
+                  id="driveHash"
+                  class="form-control"
+                  type="text"
+                  readonly
+                >
+              </div>
+              <div class="form-group">
+                <label for="confirmHash">{{ $t('drive_form.confirm_hash') }}</label>
```

Comment:
`drive_form.confirm_hash` is missing translation in `translations.json` resulting in this:
![image](https://user-images.githubusercontent.com/8676740/69713651-34689800-1105-11ea-9d6a-632adea0bd82.png)